File: src/main/java/com/jewelexx/tablocation/github/Tags.java
package com.jewelexx.tablocation.github;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Tags {
    private String name;
    private String zipballURL;
    private String tarballURL;
    private Commit commit;
    private String nodeID;

    @JsonProperty("name")
    public String getName() {
        return name;
    }

    @JsonProperty("name")
    public void setName(String value) {
        this.name = value;
    }

    @JsonProperty("zipball_url")
    public String getZipballURL() {
        return zipballURL;
    }

    @JsonProperty("zipball_url")
    public void setZipballURL(String value) {
        this.zipballURL = value;
    }

    @JsonProperty("tarball_url")
    public String getTarballURL() {
        return tarballURL;
    }

    @JsonProperty("tarball_url")
    public void setTarballURL(String value) {
        this.tarballURL = value;
    }

    @JsonProperty("commit")
    public Commit getCommit() {
        return commit;
    }

    @JsonProperty("commit")
    public void setCommit(Commit value) {
        this.commit = value;
    }

    @JsonProperty("node_id")
    public String getNodeID() {
        return nodeID;
    }

    @JsonProperty("node_id")
    public void setNodeID(String value) {
        this.nodeID = value;
    }
}

File: src/main/kotlin/dev/cordor/tablocation/github/Tags.kt
package dev.cordor.tablocation.github

import com.fasterxml.jackson.annotation.JsonProperty

class Tags {
    @get:JsonProperty("name")
    @set:JsonProperty("name")
    var name: String? = null

    @get:JsonProperty("zipball_url")
    @set:JsonProperty("zipball_url")
    var zipballURL: String? = null

    @get:JsonProperty("tarball_url")
    @set:JsonProperty("tarball_url")
    var tarballURL: String? = null

    @get:JsonProperty("commit")
    @set:JsonProperty("commit")
    var commit: Commit? = null

    @get:JsonProperty("node_id")
    @set:JsonProperty("node_id")
    var nodeID: String? = null
}
